Rick-and-Morty/Rick And Morty/Views/CharacterCell.swift

```diff
@@ -26,17 +26,26 @@ struct CharacterCell: View {
                 
                 VStack(alignment: .leading, spacing: 8) {
                     Text(character.name)
-                    Text(character.description)
+                    Text(description(for: character))
                 }
                 
                 if imagePosition == .right {
                     CharacterCellImage(character: character)
                 }
+                Spacer()
             }
             .padding()
         }
         .buttonStyle(PlainButtonStyle())
     }
+    
+    func description(for character: Character) -> String {
+        if let c = character as? ShortCharacterDescription {
```

Comment:
Try not to use one letter variables, Usually there will be setting in swift lint that will prevent you from that. you can use `char` or. event `character.